### ISSUE TITLE
[BUG] VideoSourceNotFound and unable to catch

### ISSUE THREAD

**Issue by Timtam** [Mithronn/rusty_ytdl#16]
Hello,

first of all, thank you for creating rusty_ytdl, its an awesome library, not just easy to use but incredibly fast as well.

I however seem to encounter an issue that panics my released app without me being able to prevent it, as it seems.
The error reads as follows:

```
thread 'main' panicked at /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/rusty_ytdl-0.6.6/src/info.rs:414:42:                                                                                                   
called `Result::unwrap()` on an `Err` value: VideoSourceNotFound                                        
```

Sorry for using 0.6.6 here, the same error occurs on 0.6.7, I just thought downgrading could fix it because I was already able to download the video on a different machine, obviously that idea didn't came to fruition. The video which causes this error is this one: https://www.youtube.com/watch?v=BBYdvFWsFBc

Thank you!

**Comment by Mithronn** [comment]
Thanks for your great comment and reporting the bug. I released a new version (v0.7.0) of the package and it seems fixed the issue.

**Comment by rdbo** [comment]
I have this same issue using the latest master commit (85d6d332987abf7f8860dcabe33783c9ee58b7c1)
